uk.ac.kcl.inf.modelling.xdsml.gemoc_henshin/src/uk/ac/kcl/inf/modelling/xdsml/gemoc_henshin/engine/ui/launcher/tabs/LaunchConfigurationTab.java

```diff
@@ -7,7 +7,11 @@
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Group;
 import org.eclipse.swt.widgets.Label;
-
+/**
+ * 
+ * Class fully taken from the sequential engine(Zschaler)

```

Comment:
Remove. Explain what this does. Add an issue to relevant GEMOC repo.

Reply:
i don't really remember, what's the issue in here?

Reply:
Why we had to copy the code from the original GEMOC repo rather than being able to inherit it / directly use their class.